Platform/CoffeelakeBoardPkg/Library/Stage1ABoardInitLib/Stage1ABoardInitLib.c

```diff
@@ -160,6 +163,24 @@ BoardInit (
     if (!EFI_ERROR(Status)) {
       SetCurrentBootPartition (BootPartition == BootPartition2 ? 1 : 0);
     }
+
+    // Enlarge the code cache region to cover full flash for non-BootGuard case only
+    if ((AsmReadMsr64 (MSR_BOOT_GUARD_SACM_INFO) & B_BOOT_GUARD_SACM_INFO_NEM_ENABLED) == 0) {
+       // WHL FSP-T does not allow to enable full flash code cache due to cache size restriction.
+       // Here, MTRR is patched to enable full flash region cache to avoid performance penalty.
+       // However, the SBL code flow should ensure only limited flash regions will be accessed
+       // before FSP TempRamExit() is called. The combined DATA and CODE cache size should satisfy
+       // the BWG requirement.
```

Comment:
By BWG, do you mean BIOS Writer's Guide?

Reply:
Yes, it is.